### PR TITLE
Connect matching service front end with server

### PR DESCRIPTION
**Pull request by gohyongjing**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "dev:auth": "cd microservices/auth && npm run dev",
     "dev:user-service": "cd microservices/user-service && npm run start",
     "dev:questions-service": "cd microservices/questions-service && npm run dev",
-    "dev-all": "concurrently -c \"auto\" \"npm:dev:*\""
+    "dev:rabbitMQ": "cd microservices/matching-service && npm run start:rabbitMQ",
+    "dev:matching-service": "cd microservices/matching-service && npm run dev",
```
Comment (rexcyrio) [inline package.json:18]: Do we need an install script for the `matching-service` too?

File: package.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
...
+    "dev-all": "concurrently -c \"red,green,yellow,blue,magenta,cyan,gray\" \"npm:dev:*\""
```
Comment (rexcyrio) [inline package.json:19]: We can change the `install-all` npm script to use the same colours too

**Review by rexcyrio — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gohyongjing) [reply package.json:19]: Fixed in [efc7183](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g37/pull/21/commits/efc71839b5b087bd24a8e1f3e04f623501b7be4b)

File: package.json (hunk 2 above)
Comment (gohyongjing) [reply package.json:20]: concurrently was using white as one of the colours, which is not distinguishable from the log messages, so i excluded white from the list of colours

Fixed in [efc7183](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g37/pull/21/commits/efc71839b5b087bd24a8e1f3e04f623501b7be4b)

**Review by rexcyrio — APPROVED** [review]
LGTM :+1:

**Pull request merged**